fhir_river_api/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.7-slim
+
+ARG FHIR_RIVER_API_HOME=/test
```

Comment:
Not important but why is the folder called test?

Reply:
haha, I just wanted to make sure it had no effect